src/hotspot/share/gc/serial/serialStringDedup.inline.hpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021, Alibaba Group Holding Limited. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+#ifndef SHARE_GC_SERIAL_STRINGDEDUP_INLINE_HPP
+#define SHARE_GC_SERIAL_STRINGDEDUP_INLINE_HPP
+
+#include "gc/serial/serialHeap.hpp"
+#include "gc/serial/serialStringDedup.hpp"
```

Comment:
Inline headers are required to include the associated .hpp first.  That's typically with a blank line between it and the remainder of the includes for emphasis.
https://github.com/openjdk/jdk/blame/ddcd851c43aa97477c7e406490c0c7c7d71ac629/doc/hotspot-style.md#L141-L144

Reply:
fixed.